src/main/scala/sigmastate/eval/RuntimeCosting.scala

```diff
@@ -814,6 +814,9 @@ trait RuntimeCosting extends SigmaLibrary with DataCosting with Slicing { IR: Ev
         case box: ErgoBox =>
           val boxV = liftConst(box.toTestBox(false)(IR))
           RCCostedBox(boxV, costOf(c))
+        case box: special.sigma.Box =>
```

Comment:
@greenhat This is strange. In which example this case is required?

Reply:
In array ops tests in `TestingInterpreterSpec` (access box in env). It will be resolved by #325 so I rewrote tests to avoid using a box ([see](
https://github.com/ScorexFoundation/sigmastate-interpreter/pull/317/commits/ffc1048c03fdb55c9f34c4737c55b5fb8521e625)).